docs/data-management-and-versioning.md

```diff
@@ -161,8 +207,21 @@ Storing plots is useful to visualize the data and to show any information regard
 For example, the following code creates a confusion matrix plot using [sklearn.metrics.plot_confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.plot_confusion_matrix.html#sklearn.metrics.plot_confusion_matrix) and stores the plot in the artifact repository:
 
 ``` python
-    cmd = metrics.plot_confusion_matrix(model, xtest, ytest, normalize='all', values_format='.2g', cmap=plt.cm.Blues)
-    context.log_artifact(PlotArtifact('confusion-matrix', body=cmd.figure_), local_path='plots/confusion_matrix.html')
+from mlrun.artifacts import PlotArtifact
+from mlrun.mlutils import gcf_clear
+
+gcf_clear(plt)
+cmd = metrics.plot_confusion_matrix(model, xtest, ytest, normalize='all', values_format = '.2g', cmap=plt.cm.Blues)

```

Comment:
what does cmd means ?
The only thing I can think of is shortcut of command which I don't understand how it's related to the context
we can probably find a better naming

Reply:
I took it from mlutils.eval_class_mode. I'll change the name here so it's clearer (likely the original meaning is confusion matrix diagram)